### PR TITLE
bugfix picasso resize new entry

### PR DESCRIPTION
**Pull request by pmattmann**
found the reason, why each time one try to create a new activity, it is placed 15 min later :)

### CODE REVIEW

File: frontend/src/components/program/picasso/useDragAndDropNew.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export function useDragAndDropNew(enabled, createEntry) {
 
   // resize placeholder entry
   const resizeEntry = (entry, mouse) => {
-    const { min, max } = minMaxTime(mouse, mouseStartTimestamp)
+    const { min, max } = minMaxTime(mouseStartTimestamp, mouse)
```
Comment (pmattmann) [inline frontend/src/components/program/picasso/useDragAndDropNew.js:43]: Signature of minMaxTime is 
`minMaxTime(start, end) => { min: ..., max:... }`

**Review by carlobeltrame — COMMENTED** [review]
Nice, but now the same problem happens when I drag upwards. Surely this is a less common use case. But could you maybe simply pass the smaller of the two as the first argument and the larger of the two as the second?

**Comment by BacLuc** [comment]
Cool, i think the proposal of @carlobeltrame is worth a shot

**Comment by pmattmann** [comment]
> Nice, but now the same problem happens when I drag upwards. Surely this is a less common use case. But could you maybe simply pass the smaller of the two as the first argument and the larger of the two as the second?

I'm not able to reproduce this problem.
Be aware, the displayed activity has a minimal height measured from the top. Therefore very short activities created by moving up look a bit strange - but are correct, as far as i can see it.

**Comment by BacLuc** [comment]
> > Nice, but now the same problem happens when I drag upwards. Surely this is a less common use case. But could you maybe simply pass the smaller of the two as the first argument and the larger of the two as the second?
> 
> I'm not able to reproduce this problem. Be aware, the displayed activity has a minimal height measured from the top. Therefore very short activities created by moving up look a bit strange - but are correct, as far as i can see it.

I could reproduce the issue.
Not always, but its possible.

On dev:
[Screencast from 15.04.2023 11:48:35.webm](https://user-images.githubusercontent.com/1506818/232206599-8080856b-7dfe-4134-a09f-6eedce9f3847.webm)


On PR deployment:
[Screencast from 15.04.2023 11:44:37.webm](https://user-images.githubusercontent.com/1506818/232206617-5653e02e-e7e3-4b40-8017-33caff67f5d6.webm)

You don't see it well, but to me it looks that the behaviour was inverted.
On dev it jumps down 15 minutes if you draw the ScheduleEntry down, but works well if you draw it up.
On the pr deployment the behavior is inverted.

But i rather have the behavior in the PR than the one on devel.

**Review by usu — APPROVED** [review]

**Comment by usu** [comment]
Superseded by #3468

**Pull request closed without merging**